tests/indico_wrapper/test_modelops.py

```diff
@@ -0,0 +1,29 @@
+"""
+Test Model Ops class methods
```

Comment:
Up to you, and it can be tough when external API calls are involved, but I think some more comprehensive tests of all methods would be helpful here - like testing for getting model settings, returning most recent vs. selected id. That way, if for example the return data changes in a future version, that will be apparent if the tests are rerun.